src/Razor/src/Microsoft.CodeAnalysis.Razor.Workspaces/Formatting/Passes/New/CSharpFormattingPass.cs

```diff
@@ -113,6 +142,16 @@ public async Task<ImmutableArray<TextChange>> ExecuteAsync(FormattingContext con
             {
                 iFormatted++;
             }
+            else if (lineInfo.SkipNextLineIfBrace)
+            {
+                // If the next line is a brace, we skip it, otherwise we don't. This is used to skip the opening brace of a class
+                // that we insert, but Roslyn settings might place on the same like as the class declaration.
+                if (iFormatted + 1 < formattedCSharpText.Lines.Count &&
+                    formattedCSharpText.Lines[iFormatted + 1].CharAt(0) == '{')
```

Comment:
>formattedCSharpText.Lines[iFormatted + 1] [](http://example.com/codeflow?start=20&length=41)

I haven't read the rest of this method closely, just wanted to make sure that we're confident this line isn't empty

Reply:
Good call